### PR TITLE
British columbia age credit

### PR DESCRIPTION
**Pull request by PavelMakarchuk**
Fixes #143

### CODE REVIEW

File: policyengine_canada/variables/gov/provinces/bc/tax/income/credits/bc_age_credit/bc_age_credit.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    def formula(person, period, parameters):
+        age = person("age", period)
+        p = parameters(
+            period
+        ).gov.provinces.bc.tax.income.credits.bc_age_credit
+        eligible = age >= p.eligible_age
+        income = person("total_individual_pre_tax_income", period)
+        base = p.base
+        reduction = p.phase_out.rate.calc(income)
+        return eligible * (max_(0, base - reduction))
```
Comment (MaxGhenis) [inline policyengine_canada/variables/gov/provinces/bc/tax/income/credits/bc_age_credit/bc_age_credit.py:20]:
```suggestion
        return eligible * (max_(0, p.base - reduction))
```

File: policyengine_canada/parameters/gov/provinces/bc/tax/income/credits/bc_age_credit/base.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+description: British Columbia age credit base amount.
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/bc/tax/income/credits/bc_age_credit/base.yaml:1]: could you rename the folder `age`

File: policyengine_canada/variables/gov/provinces/bc/tax/income/credits/bc_age_credit/bc_age_credit.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+from policyengine_canada.model_api import *
+
+
+class bc_age_credit(Variable):
+    value_type = float
+    entity = Person
+    label = "British Columbia Age credit, non-refundable"
+    unit = CAD
+    definition_period = YEAR
```
Comment (MaxGhenis) [inline policyengine_canada/variables/gov/provinces/bc/tax/income/credits/bc_age_credit/bc_age_credit.py:9]: Alternatively, split this into `bc_age_credit_eligible` and use that in `defined_for`
```suggestion
    definition_period = YEAR
    defined_for = ProvinceCode.BC
```

**Review by MaxGhenis — APPROVED** [review]

File: policyengine_canada/parameters/gov/provinces/bc/tax/income/credits/bc_age_credit/base.yaml (hunk 2 above)
Comment (PavelMakarchuk) [reply policyengine_canada/parameters/gov/provinces/bc/tax/income/credits/bc_age_credit/base.yaml:1]: Should I rename the entire "bc_age_credit" folder or just one specific file?

Comment (MaxGhenis) [reply policyengine_canada/parameters/gov/provinces/bc/tax/income/credits/bc_age_credit/base.yaml:1]: folder. in general we don't need parameter folders to restate other parts of folder tree

Comment (PavelMakarchuk) [reply policyengine_canada/parameters/gov/provinces/bc/tax/income/credits/bc_age_credit/base.yaml]: Ok thank you! I have also renamed the tax reduction folder.

**Pull request merged**